pyhealth/models/base_model.py

```diff
@@ -13,7 +13,7 @@
 
 
 # TODO: add support for regression
-VALID_MODE = ["binary", "multiclass", "multilabel"]
+VALID_MODE = [None, "binary", "multiclass", "multilabel"]
```

Comment:
Could we replace "None" with "Ranking" or something else?

Reply:
The evaluation pipeline for ranking is not compatible with Pyhealth's Trainer. That's why I add a None option here. I will add more instructions.